src/DotNetPad/DotNetPad.Applications/CodeAnalysis/TemplateCode.cs

```diff
@@ -2,47 +2,59 @@
 {
     internal static class TemplateCode
     {
+        public static string StartCaretIndicator
+        {
+            get
+            {
+                return "@StartCaretPosition@";
+            }
+        }
+
         public static string InitialCSharpCode
         {
             get
             {
                 return
                     @"using System;
+using System.Collections.Generic;
 using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.IO;

```

Comment:
It's also why `System.IO` and `System.Text` are imported, it allows you to quickly transform files of various encodings.